types/node-zopfli-es/index.d.ts

```diff
@@ -0,0 +1,70 @@
+// Type definitions for node-zopfli-es 0.0
+// Project: https://github.com/jaeh/node-zopfli-es
+// Definitions by: Alorel <https://github.com/Alorel>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```

Comment:
Duplicate line

Reply:
Oops. Fixed in 13b193e9ec7ffb0f4539155e3af3e7106396fee5